.changeset/twenty-onions-check.md

```diff
@@ -0,0 +1,5 @@
+---
+'@builder.io/qwik': minor
+---
+
+Add sponsors to the readme sponsor section
```

Comment:
is the changeset required here? I think this change is not related to the qwik core/city or other packages